service/src/main/java/uk/gov/hmcts/reform/fpl/Application.java

```diff
@@ -13,7 +13,7 @@
     "uk.gov.hmcts.reform.fnp.client",
     "uk.gov.hmcts.reform.calendar.client"
 })
-@ComponentScan
+@ComponentScan(basePackages = {"uk.gov.hmcts.reform.fpl", "uk.gov.hmcts.reform.testingsupport"})
```

Comment:
i think if package is uk.gov.hmcts.reform.fpl.testingsupport then we dont need and probably is better anyway as this is fpl specific